src/main/scala/poker/showdowns/Event.scala

```diff
@@ -0,0 +1,68 @@
+package poker.showdowns
+
+import cats.implicits.catsSyntaxEitherId
+import cats.kernel.Monoid
+import org.scalactic.anyvals.NonEmptySet
+import poker.{Player, Position, PreflopDeck, ShowDown, WinnerList}
+import cats.kernel.Monoid
+
+import java.time.LocalDateTime
+
+sealed trait Event[A] {
+  def timeStamp: LocalDateTime
+}
+
+case class PreFlopShownDown(
+  winners: WinnerList,
+  timeStamp: LocalDateTime = LocalDateTime.now()
```

Comment:
Referentially opaque. I recommend using an envelope instead and pushing this into infrastructure.

Reply:
If we could discuss this in more detail that would be great.  I'm definitely having some issues modeling this.  

And I accidentally mixed this into this pr.  I will try to get it out of here.